tests/Fractions.Tests/FractionSpecs/FromDoubleRounded/Method_FromDoubleRounded.cs

```diff
@@ -210,4 +211,115 @@ public class When_a_fraction_is_created_with_1_third : Spec {
     public void The_fraction_should_then_have_a_denominator_of_3() {
         _fraction.Denominator.Should().Be(3);
     }
-}
+}
+
+[TestFixture]
+public class When_a_fractions_is_created_by_rounding_a_double_without_precision : Spec {
+    private const double DoubleValue = 1055.05585262;
+    private const decimal LiteralValue = 1055.05585262m; // the "true/literal" decimal representation
+
+    private Fraction _fraction;
+
+    public override void Act() {
+        _fraction = Fraction.FromDoubleRounded(DoubleValue);
+    }
+
+    [Test]
+    public void The_actual_fraction_may_differ_from_the_literal_value() {
+        LiteralValue.Should().Be((decimal)DoubleValue).And.NotBe(_fraction.ToDecimal());

```

Comment:
I don't think this is the _obvious_- perhaps we should add some more remarks to the XML docs.

Reply:
I thought that would explain the facts exactly:  :sweat_smile:

![image](https://github.com/danm-de/Fractions/assets/3825484/a62593cc-8b22-4307-8db2-5a5de7252630)

What would you add?

Reply:
Added

```csharp
    [Test]
    public void The_actual_fraction_is_an_approximation_of_the_literal_value_as_a_decimal() {
        _fraction.ToDecimal().Should().BeApproximately(LiteralValue, 8);
    }
```